vault/core.go

```diff
@@ -3234,28 +3235,7 @@ func (c *Core) isMountable(ctx context.Context, entry *MountEntry, pluginType co
 // isMountEntryBuiltin determines whether a mount entry is associated with a
 // builtin of the specified plugin type.
 func (c *Core) isMountEntryBuiltin(ctx context.Context, entry *MountEntry, pluginType consts.PluginType) bool {
```

Comment:
If you're going to have separate implementations for OSS/ENT, you'd want to do this in `core_util.go`.

Reply:
not sure how this happened - but removed this impl because this wasn't a change in ent either.